### DISCUSSION TITLE
Ver source code...

### DISCUSSION THREAD

**Discussion by mmasias** [TheMoys/24-25-IdSw1-SDR#84] [Ideas]
<table>
<tr>
<th>

### 🔰 Modelo de Dominio
</th>
<th>

### 🔰 Disciplina de Requisitos

</th>
</tr>
<tr></tr>
  <tr>
    <td valign=top>
      <p>El <a href="docs/modeloDeDominio/README.md">modelo de dominio</a> de <em>DOCUTRACE</em> define las entidades principales y sus relaciones dentro del sistema.</p>
      <p>Para describir el modelo del dominio se utilizan los siguientes artefactos:</p>
      <ul>
        <li><a href="docs/modeloDeDominio/README.md#diagrama-de-clases">Diagrama de Clases</a></li>
        <li><a href="docs/modeloDeDominio/README.md#diagrama-de-objetos">Diagrama de Objetos</a></li>
        <li><a href="docs/modeloDeDominio/README.md#diagrama-de-estados">Diagrama de Estados</a></li>
      </ul>
    </td>
<td valign=top>
      <p>En esta sección se describen los actores principales que interactúan con el sistema y los casos de uso que representan las funcionalidades clave del proyecto.</p>
      <p>Se pueden ver más detalles sobre esto con las siguientes herramientas:</p>
      <ul>
        <li><a href="docs/casosDeUso/actores/README.md">Actores</a></li>
        <li><a href="docs/casosDeUso/diagramaDeContexto/README.md">Diagrama de Contexto</a></li>
        <li><a href="docs/casosDeUso/diagramaCasosDeUso/README.md">Diagrama de Casos de Uso</a></li>
        <li><a href="docs/casosDeUso/detalladoCasosDeUso/README.md">Detallado de Casos de Uso</a></li>
        <li><a href="docs/casosDeUso/prototipos/README.md">Prototipos</a></li>
      </ul>
    </td>
  </tr>
</table>

**Comment by TheMoys** [comment]
Gracias!!! Es lo que estábamos buscando para ordenarlo y que se vea bien 😄

**Comment by TheMoys** [comment]
@mmasias 
Para la presentación de mañana escogerás tú a uno que expondrá? Y de ser así, que formato tendrá la exposición? Como si lo estuviésemos presentando desde cero o irás haciendo preguntas puntuales en cada apartado?

**Reply by mmasias** [reply]
Formato reunión de trabajo. Y, sí: iré eligiendo quién responde.

**Reply by TheMoys** [reply]
Haciendo una revisión a profundidad nos hemos topado con una incógnita, el detallado de casos de uso que corresponden al actor tiempo se deben de hacer? Y de ser así de que manera se representa? Ya que no hay interacción de un actor viviente

**Comment by mmasias** [comment]
El actor ataca, con lo cual el actor tiempo es el que dispara la acción.
Probablemente cause extrañeza el que no haya "interfaz" de usuario como
tal, con lo cual és algo así como: el actor tiempo solicita generar un
informe de bla bla bla para enviarlo a X persona, el sistema genera el
informe que contiene tal y tal y tal dato

Si te fijas, la trama de conversación es la interface: el tipo de informe y
el destinatario

Queda?


-
Manuel Masías
http://es.linkedin.com/in/mmasias

**Reply by TheMoys** [reply]
Entendido! Gracias!

**Reply by mmasias** [reply]
Ah, y solo por higiene: pregunta nueva, hilo nuevo. Si no, es como los comentarios en programación... Duda resuelta, se cierra el hilo.
